openstack/sharedfilesystems/v2/shares/results.go

```diff
@@ -110,3 +110,39 @@ type DeleteResult struct {
 type GetResult struct {
 	commonResult
 }
+
+// GrantAccessRes contains all information associated with an OpenStack share Grant Access
+type GrantAccessRes struct {
```

Comment:
I renamed the `func GrantAccess` to `func GrantAccessRequest` and the `type GrantAccessRes` to `type GrantAccess`.